routes/ci.js

```diff
@@ -0,0 +1,126 @@
+var express = require("express");
+var router = express.Router();
+var request = require("request")
+var repositoryModel = require("../model/repository.js");
+var crypter = require("../util/crypter.js");
+var generator = require("../backend/generator.js");
+var deploy = require("../backend/deploy.js");
+router.get('/register', function (req, res, next) {
+  request({
+    url: `https://api.github.com/users/${req.session.username}/repos?per_page=100`,
+    headers: {
+      'User-Agent': 'request'
+    }
+  },
+  function (error, response, body) {
+    var repos = JSON.parse(body);
+    res.render("index", {
```

Comment:
I think we should keep the index template specific for the `/` route. What is happening here is that since the modal is collapsible, i can perform the actions which are aesthetically supposed to be performed in the `/` route, at the `/ci/register` route.